manifests/base/deployment/argocd-image-updater-deployment.yaml

```diff
@@ -77,6 +77,24 @@ spec:
               name: argocd-image-updater-config
               key: git.email
               optional: true
+        - name: GIT_COMMIT_SIGNING_KEY
+          valueFrom:
+            configMapKeyRef:
+              key: git.commit-signing-key
+              name: argocd-image-updater-config
+              optional: true
+        - name: GIT_COMMIT_SIGNING_METHOD
+          valueFrom:
+            configMapKeyRef:
+              key: git.commit-signing-key
```

Comment:
should this be git.commit-signing-method?

Reply:
yes, just pushed an update!